src/org/labkey/test/components/ui/entities/EntityUpdateFromFilePanel.java

```diff
@@ -0,0 +1,55 @@
+package org.labkey.test.components.ui.entities;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.ui.grids.ResponsiveGrid;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.File;
+
+public class EntityUpdateFromFilePanel extends EntityInsertPanel
```

Comment:
Does this really need to extend the EntityInsertPanel? Unless I'm missing something it doesn't look like it needs all of the functionality of EntityInsertPanel, and can be a fairly simple panel on its own.